vendor.conf

```diff
@@ -27,7 +27,7 @@ github.com/imdario/mergo                            7c29201646fa3de8506f70121347
 golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
 
 # buildkit
-github.com/moby/buildkit                            8c0fa8fdec187d8f259a349d2da16dc2dc5f144a # v0.5.0
+github.com/moby/buildkit                            f238f1efb04f00bf0cc147141fda9ddb55c8bc49
```

Comment:
Will there be a `v0.5.2` tag for this, or is more still coming?

Reply:
We still have 2 open issues in v0.5.2 milestone. At least one of them is unrelated to moby.

Reply:
ok; we can do one more bump before GA, and then tag 👍